README.md

```diff
@@ -1,10 +1,13 @@
-## Docker for Visual Studio Code
 
-[![Version](https://vsmarketplacebadge.apphb.com/version/ms-azuretools.vscode-docker.svg)](https://marketplace.visualstudio.com/items?itemName=ms-azuretools.vscode-docker) [![Installs](https://vsmarketplacebadge.apphb.com/installs-short/ms-azuretools.vscode-docker.svg)](https://marketplace.visualstudio.com/items?itemName=ms-azuretools.vscode-docker) [![Build Status](https://dev.azure.com/ms-azuretools/AzCode/_apis/build/status/Nightly/vscode-docker-nightly-2?branchName=master)](https://dev.azure.com/ms-azuretools/AzCode/_build/latest?definitionId=22&branchName=master)
+The Docker extension makes it easy to build, manage, and deploy containerized applications from Visual Studio Code. It also provides one-click debugging for Node.js, Python, and .NET Core.
```

Comment:
> It also provides one-click debugging for Node.js, Python, and .NET Core.

It is not very clear from this sentence that the debugging capability is about **debugging inside a container**.

Reply:
Updated to mentnion "inside a container"